### ISSUE TITLE
duration is null in summary.json file

### ISSUE THREAD

**Issue by grawk** [krakenjs/nemo#12]
```
{
	"totals": {
		"total": 12,
		"pass": 8,
		"fail": 4
	},
	"instances": [
		{
			"tags": {
				"profile": "chrome",
				"grep": "@p2",
				"key": "bing",
				"reportFile": "/03-26-2018/13-58-09/profile!chrome!grep!@p2!key!bing/nemo-report.html",
				"uid": "39746095-3cec-4cb0-b3bb-89655cfab133"
			},
			"duration": null,
			"summary": {
				"total": 1,
				"pass": 1,
				"fail": 0
			}
		},
		{
...
```

**Comment by grawk** [comment]
should be fixed by #34

**Comment by grawk** [comment]
Fixed in v4.7.0
